plugins/Alias/config.py

```diff
@@ -43,5 +43,7 @@ def configure(advanced):
 Alias = conf.registerPlugin('Alias')
 conf.registerGroup(Alias, 'aliases')
 conf.registerGroup(Alias, 'escapedaliases')
+conf.registerGlobalValue(Alias, 'validName',
+    registry.String(r'^[^\x00-\x20]+$', _("""Regex which alias names must match in order to be valid""")))
```

Comment:
You should make it more restrictive by using a whitelist instead of a blacklist.

Also, making this configurable makes it very easy for bot owners to break their bot + config file.